plugins/pencilblue/controllers/admin/sites/site_form.js

```diff
@@ -75,22 +75,36 @@ module.exports = function SiteFormModule(pb) {
     var options = {isNew: true};
     //todo:: refactor angular on site_form to remove unneeded properties
     setupAngularObj(self, options, cb);
-
   };
 
   function setupAngularObj(self, options, cb){
     var isNew = options.isNew,
       display = options.display,
       host = options.host,
+      supportedLocales = {},
+      savedLocales = options.savedLocales || {},
+      selectedLocales = [],
+      defaultLocale = options.defaultLocale || "",
       isActive = options.isActive,
       uid = options.uid;
 
+    for (var locale in pb.Localization.supportedLookup) {
```

Comment:
Can we use ```Localization.getSupported()``` instead?  It will return an array of locale strings. Then you can simplify things a bit through:

```
selectedLocales = pb.Localization.getSupported().filter(function(locale) {
  return !!savedLocales[locale];
});
```

I'm not sure I understand the difference between savedLocales and selected locales.